package.json

```diff
@@ -136,12 +138,13 @@
     "testcafe-browser-tools": "2.0.23",
     "testcafe-hammerhead": "24.5.18",
     "testcafe-legacy-api": "5.1.4",
-    "testcafe-reporter-dashboard": "0.2.5",
+    "testcafe-reporter-dashboard": "0.2.6-rc.3",
```

Comment:
Despite the common case we also have a situation when user has already registered in Dashboard and they use `testcafe dashboard` command. We should have text that is suitable for both cases: registration and signing-in.
E.g. the second phrase can be `Copy a token from Dashboard to set as a default token on your machine. Or complete the process by pressing any key.`

Reply:
@titerman 
If the case isn't quite clear please text me :)